rust/core-lib/src/find.rs

```diff
@@ -120,14 +120,18 @@ impl Find {
     }
 
     /// Sets find parameters and search query.
-    fn set_find(&mut self, search_string: &str,
-                case_sensitive: bool) {
-        let case_matching = if case_sensitive {
-            CaseMatching::Exact
+    fn set_find(&mut self, search_string: &str, case_sensitive: bool, regex: bool) {
+        let case_matching = if regex {
+            CaseMatching::RegularExpression
```

Comment:
This is related to another comment - I'm wondering why we're throwing away case_sensitive in the regex case, because it is a very reasonable option to pass along to regex.